cmd/discovery.go

```diff
@@ -89,11 +89,17 @@ func discoverNodesForService(serviceName string) ([]esnode, error) {
 
 	var nodeList []esnode
 	for _, svc := range catalogServices {
-		log.Debug("Service discovered: ", svc.Node, " (", svc.Address, ":", svc.ServicePort, ")")
+		var addr string
+		if svc.ServiceAddress != "" {
+			addr = svc.ServiceAddress
+		} else {
+			addr = svc.Address
```

Comment:
can't we do something like
var addr string = svc.Address
		if svc.ServiceAddress != "" {
			addr = svc.ServiceAddress
		}

Reply:
We can do that